src/decode.ts

```diff
@@ -208,11 +248,7 @@ export function decode(qr: string): DataModel {
 	const bytes = base32hex.decode(qr);
 	const bysquareHeader = bytes.slice(0, 2);
 	const decodedBysquareHeader = bysquareHeaderDecoder(bysquareHeader);
-	if ((decodedBysquareHeader.version > Version["1.1.0"])) {
-		throw new Error(
-			`Unsupported Bysquare version '${decodedBysquareHeader.version}' in header detected. Only '0' and '1' values are supported`,
-		);
-	}
```

Comment:
I try to avoid extracting small pieces of code unless it's absolutely necessary. I know it can be tempting to break things down into smaller chunks, IMHO it often makes the code harder to read.

Reply:
[0ab4f90](https://github.com/xseman/bysquare/pull/50/commits/0ab4f90c8fc0cbf2b60971c5c55cfc46a2e48997)